src/EFCore/Query/QueryableMethodTranslatingExpressionVisitor.cs

```diff
@@ -54,6 +54,29 @@ public abstract class QueryableMethodTranslatingExpressionVisitor : ExpressionVi
         /// </summary>
         protected virtual QueryableMethodTranslatingExpressionVisitorDependencies Dependencies { get; }
 
+        /// <summary>
+        ///     Additioanl information about errors encountered during translation.
+        /// </summary>
+        public virtual string TranslationErrorDetails { get; private set; }
+
+        /// <summary>
+        ///     Method which allows providing additional information about errors encountered during translation.
```

Comment:
```suggestion
        ///     Provides additional information about errors encountered during translation.
```
Though I think the name of this API should be changed. But we will discuss in API review.